survey/dashboards/dashboard.py

```diff
@@ -0,0 +1,21 @@
+from django_plotly_dash import DjangoDash
+import dash_bootstrap_components as dbc
+from .layouts import create_dashboard_layout
+from .callbacks import register_callbacks
+from survey.models import SurveyResponse
+
+survey_data = list(SurveyResponse.objects.all().values())
```

Comment:
Just noticed this but this call essentially gets all response data from our entire database, all responses from different surveys will be mixed together!